src/features/userSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+export type UserState = {
+  users: User[];
+};
+
+const initialState: UserState = {
+  users: [],
+};
```

Comment:
Consider initializing `loading` and `err` states in the `initialState` to manage loading and error states during the fetch operation.